lib/windows/winlean.nim

```diff
@@ -1129,5 +1133,41 @@ proc setFileTime*(hFile: Handle, lpCreationTime: LPFILETIME,
                  lpLastAccessTime: LPFILETIME, lpLastWriteTime: LPFILETIME): WINBOOL
      {.stdcall, dynlib: "kernel32", importc: "SetFileTime".}
 
+type
+  SID_IDENTIFIER_AUTHORITY* {.importc, header: "windows.h".} = object
+    Value*: array[6, BYTE]
+  PSID_IDENTIFIER_AUTHORITY* {.importc, header: "windows.h".} = ptr SID_IDENTIFIER_AUTHORITY
```

Comment:
why not:
```suggestion
  PSID_IDENTIFIER_AUTHORITY* = ptr SID_IDENTIFIER_AUTHORITY
```

but furthermore, why do we need it, can't we just use `ptr PSID_IDENTIFIER_AUTHORITY` in the only place where this is used?